app/student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom, id, age, name, parent_permission)
```

Comment:
- [ ] Good job so far, but I think the name has a value `Unknown` and the parent_permission should be set up to `true`. You don't need to pass the `id` as an argument 🙃